system/types/settings.go

```diff
@@ -38,6 +39,7 @@ type (
 )
 
 const (
+	passwordMinLength        = 8
```

Comment:
What is the reason that we're enforcing this?

Reply:
We have a default min-length for password is 8.